todo_app/app.py

```diff
@@ -12,5 +12,10 @@ def index():
 
 @app.route('/new', methods=['GET', 'POST'])
 def create():
-    add_item(request.form.get('title'))
+    add_item(request.form.get('name'))
+    return redirect('/')
+
+@app.route('/resolve/<id>')
+def resolve(id):
+    resolve_item(id)
     return redirect('/')
```

Comment:
These should both be `POST` only ideally - GET requests shouldn't change state on the server, it doesn't play nice with browser behaviour (e.g. confirm form resolution confirm box, security-wise - cross origin GET requests are much more relaxed, caching).

You'll need to change your HTML to use a form and submit button rather than an `<a>` tag